RadixWallet/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -23,16 +23,11 @@ final actor ProfileStore {
 }
 
 extension ProfileStore {
-	func profile() async -> Profile {
-		try! await profileSubject.first()
```

Comment:
This implementation would crash the app if parent task was cancelled, as here a force unwrap would be done when a CancellationError is thrown. Instead, `profileSubject` is transformed to `AsyncCurrentValueSubject<Profile?>` to be able to access the current profile.

Reply:
I didn't do this in the first place exactly for the reasons you describe, as we don't gain anything. Storing in variable might be slightly confusing, as one would do that, if they want to potentially cancel the task at some point.